docs/integrations/destinations/snowflake.md

```diff
@@ -152,6 +152,8 @@ By default, Airbyte uses batches of `INSERT` commands to add data to a temporary
 
 Internal named stages are storage location objects within a Snowflake database/schema. Because they are database objects, the same security permissions apply as with any other database objects. No need to provide additional properties for internal staging
 
+**Operating on a stage also requires the USAGE privilege on the parent database and schema.**
```

Comment:
can you add it to the script above used to create the user?

Reply:
the script above already has needed statement 
grant OWNERSHIP
on database identifier($airbyte_database)
to role identifier($airbyte_role);